flyingcloud/base.py

```diff
@@ -761,16 +730,17 @@ def get_latest_tag(self, namespace, image_name):
         namespace.logger.info("get_latest_tag('%s') = '%s'", repo, tag)
         return tag
 
-    def docker_login(self, namespace, username, password, email=None, registry=None):
```

Comment:
@amoffat let's leave `namespace` spelled out. We should all be using autocompletion in our editors. Spelling it out makes it easier for us to understand the code.